packages/frontend/package.json

```diff
@@ -19,7 +19,8 @@
     "@unirep/core": "2.0.0-alpha-2",
     "ethers": "^5.7.2",
     "file-loader": "^6.2.0",
-    "mobx-react-lite": "^3.4.0"
+    "mobx-react-lite": "^3.4.0",
+    "react-icons": "^4.7.1"
```

Comment:
I'm not a fan of packages like this. I think we'd be better off with some custom SVGs that we import/manage ourselves.

Reply:
Doubly so because this is a template repo and we're passing on our dependencies to anyone using the repo. Better to keep dependencies minimal imo.

Reply:
This makes sense, thanks for the insight.